srcpkgs/sqlitestudio/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'sqlitestudio'
+pkgname=sqlitestudio
+version=3.3.3
+revision=1
+archs="~aarch* ~armv*"
```

Comment:
why is this necessary? If it fails to cross-compile, this is wrong and we should instead fix it (or set $nocross)